src/applications/disability-benefits/all-claims/components/Autocomplete.jsx

```diff
@@ -113,14 +124,40 @@ const Autocomplete = ({
         selectResult(results[activeIndex]);
       } else if (e.key === 'Escape') {
```

Comment:
We'll also want to add 'Tab' behavior // On Tab, user input should remain as-is, list should close, focus goes to save button.